src/clinvar_table_to_vcf.py

```diff
@@ -17,7 +17,11 @@ def gzopen(path, mode='r', verbose=True):
         return open(path, mode)
 
 
-def table_to_vcf(input_table_path, input_reference_genome, version):
+def table_to_vcf(
+        input_table_path, input_reference_genome, version,
+        # new optional/defaulted argument controls filtering of large allele strings (SAP-19901)
+        max_variant_length_thres = 7000
+    ):
```

Comment:
The limit is not 7000. That was the default value in Mastermind (which also needs to be updated).

The limit is 5900 as reported in the description of https://jira.congenica.net/browse/SAP-19901 .

Reply:
Fixed the threshold, also updated subtask description